lib/error_tracker/web/live/show.html.heex

```diff
@@ -25,6 +25,22 @@
         <%= @error.source_line %></pre>
     </.section>
 
+    <.section :if={@occurrence.breadcrumbs != []} title="Bread crumbs">
+      <div class="relative overflow-x-auto shadow-md sm:rounded-lg ring-1 ring-gray-900">
+        <table class="w-full text-sm text-gray-400 table-fixed">
+          <tr
+            :for={{breadcrumb, index} <- Enum.with_index(@occurrence.breadcrumbs)}
+            class="border-b bg-gray-400/10 border-gray-900 last:border-b-0"
+          >
+            <td class="w-11 pl-2 py-4 font-medium text-white relative text-right">
+              <%= length(@occurrence.breadcrumbs) - index %>.
+            </td>
+            <td class="px-2 py-4 font-medium text-white relative"><%= breadcrumb %></td>
+          </tr>
```

Comment:
We add breadcrumbs at the end of the list, so newest ones are the last. I think we are rendering them in reverse.

Here, the `ErrorTrackerDevWeb.Endpoint.add_breadcrumb` happens first and then `ErrorTrackerDevWeb.PageController.exception`. But the numbers are a bit confusing.

<img width="931" alt="image" src="https://github.com/user-attachments/assets/e42d0415-8e4f-4ae7-bf24-3c1ef26ba79e">

In my mind the easiest way to read this would be like a stacktrace in which newest breadcrumbs appear at the top while oldest ones appear at the bottom. Maybe we should drop the numbers?

<img width="938" alt="image" src="https://github.com/user-attachments/assets/f5a7abee-5e04-4bbc-ab28-15c22f9947e2">

To do that we could make this small change:

```suggestion
          <tr
            :for={
              {breadcrumb, index} <-
                @occurrence.breadcrumbs |> Enum.reverse() |> Enum.with_index(1)
            }
            class="border-b bg-gray-400/10 border-gray-900 last:border-b-0"
          >
            <td class="w-11 pl-2 py-4 font-medium text-white relative text-right">
              <%= index %>.
            </td>
            <td class="pl-4 px-2 py-4 font-medium text-white relative"><%= breadcrumb %></td>
          </tr>
```

Reply:
Love it!